java/src/main/java/org/rocksdb/Cache.java

```diff
@@ -10,4 +10,16 @@
   protected Cache(final long nativeHandle) {
     super(nativeHandle);
   }
+  public long getUsage() {
```

Comment:
Please add Javadoc to each public method.

Reply:
ok

Reply:
The sentence in the Javadoc description paragraph should start with a capital letter please.

Reply:
ok